src/gui/qgsstatusbarmagnifierwidget.h

```diff
@@ -0,0 +1,78 @@
+/***************************************************************************
+                         qgsstatusbarmagnifierwidget.h
+    begin                : April 2016
+    copyright            : (C) 2016 Paul Blottiere, Oslandia
+    email                : paul dot blottiere at oslandia dot com
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef QGSSTATUSBARMAGNIFIERWIDGET_H
+#define QGSSTATUSBARMAGNIFIERWIDGET_H
+
+class QLabel;
+class QFont;
+class QHBoxLayout;
+class QgsMapCanvas;
+class QgsDoubleSpinBox;
+
+#include <QWidget>
+
+/** \ingroup gui
+  * A widget which lets the user select the current level of magnification to
+  * apply to the canvas.
+  * @note added in 2.16
+  */
+class GUI_EXPORT QgsStatusBarMagnifierWidget : public QWidget
```

Comment:
@nyalldawson I think you're right. For example, the QgsStatusBarCoordinatesWidget is in app. I am going to change that.